static/app/views/performance/transactionSummary/transactionOverview/userMiseryChart/index.tsx

```diff
@@ -0,0 +1,115 @@
+import {Fragment} from 'react';
+import {withRouter, WithRouterProps} from 'react-router';
+import {Location, Query} from 'history';
+
+import EventsRequest from 'sentry/components/charts/eventsRequest';
+import {HeaderTitleLegend} from 'sentry/components/charts/styles';
+import {getInterval} from 'sentry/components/charts/utils';
+import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {t} from 'sentry/locale';
+import {Organization, OrganizationSummary} from 'sentry/types';
+import {Series} from 'sentry/types/echarts';
+import {getUtcToLocalDateObject} from 'sentry/utils/dates';
+import {useMEPSettingContext} from 'sentry/utils/performance/contexts/metricsEnhancedSetting';
+import useApi from 'sentry/utils/useApi';
+import {getTermHelp, PERFORMANCE_TERM} from 'sentry/views/performance/data';
+import {getMEPQueryParams} from 'sentry/views/performance/landing/widgets/utils';
+import {DurationChart} from 'sentry/views/performance/landing/widgets/widgets/singleFieldAreaWidget';
+
+import {ViewProps} from '../../../types';
+
+type Props = WithRouterProps &
+  ViewProps & {
+    location: Location;
+    organization: OrganizationSummary;
+    queryExtra: Query;
+    withoutZerofill: boolean;
+  };
+
+/**
+ * Fetch and render a stacked area chart that shows user misery over
```

Comment:
I don't think it's a stacked area chart in this case since we're only rendering data for a single period